src/Extensions/Hosting.Services.Web/Middlewares/UserIdentity/Options/UserIdentiyComplianceLevel.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares
+{
+	/// <summary>
+	/// Level of compliance for user identity logging
+	/// </summary>
+	public enum UserIdentiyComplianceLevel
```

Comment:
I'm wondering if it's something that should be stored in our public GitHub repo or maybe in our GitHub Enterprise repo. What do we do in other similar cases when the implementation is specific to MS?

Reply:
As far as I understand currently only two cases exist:
1. UserHash should not be traceable to the end-user after 48 hours
2. No required limitation on user hash traceability
And it's more legal than code requirements so I don't see a lot of other options appearing here. Essentially it's a switch to disable anonymizing of hashes if needed. 
And if we would get some new options I don't see problems adding them here.

Reply:
Ok, makes sense. I was just wondering if LiveId/OrgId is something that should be included here, maybe nomenclature like DataController/DataProcessor should be used instead